### PR TITLE
Create Project Scaffold quarto create project #4473

### PR DESCRIPTION
**Pull request by ghost**
#4473

### CODE REVIEW

File: src/project/types/confluence/confluence.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { ProjectScaffoldFile } from "../types.ts";
+
+export const buildConfluenceFiles = (): ProjectScaffoldFile[] => {
+  const file: ProjectScaffoldFile[] = [
+    {
+      name: "project-roadmap",
+      content: `# Welcome
+Welcome to this sample Quarto Confluence Project! Feel free to use this as a way to help get you started in building your own Confluence project.
```
Comment (allenmanning) [inline src/project/types/confluence/confluence.ts:8]: @jjallaire Would you like to see some changes here?

File: src/project/types/project-default.ts [hunk 2]
```diff
@@ -1,44 +1,58 @@
...
-    const resourceDir = resourcePath(join("projects", "default"));
     return {
       configTemplate: join(resourceDir, "templates", "_quarto.ejs.yml"),
       resourceDir,
       scaffold: (engine: string, kernel?: string, packages?: string[]) => {
-        const file: ProjectScaffoldFile[] = [{
-          name: title,
-          content: `## Quarto
+        if (template === kConfluence) {
+          return buildConfluenceFiles();
```
Comment (allenmanning) [inline src/project/types/project-default.ts:44]: @dragonstyle Would you prefer that I refactored this out in a different way?

File: src/project/types/confluence/confluence.ts (hunk 1 above)
Comment (jjallaire) [reply src/project/types/confluence/confluence.ts:8]: Maybe something more like:

"This project provides provides a simple scaffold for creating a Quarto Confluence Project. You'll almost certainly want to remove the sample files, they are just here as examples. When you add you own documents (including ones in subfolders) they will be automatically added to the site navigation sidebar."

Comment (jjallaire) [reply src/project/types/confluence/confluence.ts:8]: Also, I'm thinking that we'd want this content in the `index.qmd` ? (let's discuss realtime in our 1:1 today)

**Pull request merged**